src/simple_rules.ml

```diff
@@ -102,6 +102,7 @@ let alias sctx ?extra_bindings ~dir ~scope (alias_conf : Alias_conf.t) =
         ~f:Dune_file.Dep_conf.remove_locs alias_conf.deps
     , Option.map ~f:(fun (_loc, a) -> Action.Unexpanded.remove_locs a)
         alias_conf.action
+    , Option.map extra_bindings ~f:Pform.Map.to_stamp
```

Comment:
Perhaps more correct would be to take the subset of extra_bindings that is actually used by the alias. I'm not sure if this extra is justified though (@diml ?)

Reply:
I don't think so. AFAICS, the only extra binding is the `test` variable. If it's not used, then I don't know what the test is doing exactly...